src/iOS/toga_iOS/widgets/multilinetextinput.py

```diff
@@ -0,0 +1,129 @@
+from rubicon.objc import (
+    CGPoint,
+    NSObject,
+    objc_method
+)
+from travertino.size import at_least
+from toga_iOS.libs import (
+    NSLayoutAttributeBottom,
+    NSLayoutAttributeLeading,
+    NSLayoutAttributeTop,
+    NSLayoutAttributeTrailing,
+    NSLayoutConstraint,
+    NSLayoutRelationEqual,
+    UILabel,
+    UITextView,
+    UITextViewDelegate,
+)
+
+from .base import Widget
+
+class TogaMultilineTextView(UITextView):
+    @objc_method
+    def pointInside_withEvent_(self, point: CGPoint, event) -> bool:
+        # To keep consistency with non-mobile platforms, we'll resign the
+        # responder status when you tap somewhere else outside this view
+        # (except the keyboard)
+        within_x = point.x > 0 and point.x < self.frame.size.width
+        within_y = point.y > 0 and point.y < self.frame.size.height
+        in_view = within_x and within_y
+        if not in_view:
+            self.resignFirstResponder()
+        return in_view
+
+class MultilineTextInputDelegate(NSObject, protocols=[UITextViewDelegate]):
```

Comment:
Is there any reason the TogaMultilineTextView can't be it's own delegate?

Reply:
The protocol conformance isn't strictly required - it's nice to have, but won't actually stop anything at runtime. 

As for multiple inheritance - it doesn't need to. *All* Objective C classes are subclasses of NSObject - including UITextView. 

So, AFACIT, you should be able to add the protocol declaration to TogaMultilineTextView, and add the delegate methods to that class as well, and have everything you need.

Reply:
@freakboy3742 - got it :) I was completely overthinking it. I'll push the change, thanks for the feedback!